net/arp-scan/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=       ARP Scan
+PLUGIN_VERSION=    0.1
+PLUGIN_COMMENT=    Get all peers connected to a LAN
+PLUGIN_MAINTAINER= giuseppe.demarco@unical.it
+PLUGIN_DEPENDS=    arp-scan
+
```

Comment:
here we also need

PLUGINS_DEVEL= yes

Reply:
@fichtner typo? Shouldn't that be `PLUGIN_DEVEL`